autosklearn/pipeline/implementations/util.py

```diff
@@ -35,3 +35,12 @@ def convert_multioutput_multiclass_to_multilabel(probas):
                 multioutput_probas[:, i] = 0
         probas = multioutput_probas
     return probas
+
+def partial_mutual_info(mutual_info_func, algo_random_state):
+    def mutual_info_score(X, y, *, discrete_features='auto', n_neighbors=3,
+                          copy=True, random_state=None):
+        return mutual_info_func(X, y, discrete_features=discrete_features,
```

Comment:
Don't you need to import this somehow?

Reply:
If you are talking about `mutual_info_func ` to import, `mutual_info_func` is the method param for the `partial_mutual_info`.  We dont need to import it here.

Reply:
Yeah, got it. There was some error related while serialising/deserialising the model as the new wrapped method could not be pickled. I have updated implementation to use partials for passing random_state to mutual_info score. It works now.